webrecorder/browsertrix
# Schedule time of day when creating config

WIP https://github.com/webrecorder/browsertrix-cloud/issues/74

Formats schedule in UTC with hour and minute selections.

### Assumptions
- kube-controller-manager is in UTC
- the next weekly or monthly interval will run a week/month from template creation, not on the first day of the week or first day of the month. We could make this configurable down the road.

### Manual testing
1. Run app and go to Archives > archive > Crawl Templates > new
2. Update schedule interval and time dropdowns. Verify that "Next scheduled crawl" date updates as expected
3. Open browser dev tools network tab and save template. Verify that `schedule` in payload is formatted as expected

### Screenshots
**Default schedule field**
<img width="727" alt="Screen Shot 2022-01-17 at 6 29 58 PM" src="https://user-images.githubusercontent.com/4672952/149860647-905cfe1a-3675-44aa-a996-88e4b5e3c1db.png">

**Schedule disabled**
<img width="715" alt="Screen Shot 2022-01-17 at 6 30 08 PM" src="https://user-images.githubusercontent.com/4672952/149860664-c001bf89-c06c-4a09-b2fa-db1054ae2230.png">

**Example text near button**
<img width="477" alt="Screen Shot 2022-01-17 at 6 30 28 PM" src="https://user-images.githubusercontent.com/4672952/149860681-84e082ae-2fa5-454e-8c00-35b3df23b890.png">
